pandas/tests/io/parser/common/test_common_basic.py

```diff
@@ -862,3 +862,21 @@ def test_read_seek(all_parsers):
             actual = parser.read_csv(file)
         expected = parser.read_csv(StringIO(content))
     tm.assert_frame_equal(actual, expected)
+
+
+@pytest.mark.xfail(
+    reason="the 'pyarrow' engine does not support separators with more than one "
+    r"character, except for '\\t'. Please use a supported separator "
+    "or a different engine."
+)
```

Comment:
I'm not sure if I'm missing something, or you're misunderstanding what an xfail is. But this xfail marker is telling pytest that this test will fail, and I assume this test should pass, no?